lemur-build-docker/Dockerfile

```diff
@@ -0,0 +1,81 @@
+#############  BUILDER  #############
+FROM ubuntu:18.04 as builder
+LABEL maintainer="Netflix Open Source Development <talent@netflix.com>"
+
+ARG DEBIAN_FRONTEND=noninteractive
+ARG VERSION
+
+ARG URLCONTEXT
+
+COPY lemur/ /opt/lemur
+WORKDIR /opt/lemur
+
+RUN apt-get update && \
+    apt-get -y upgrade && apt-get install -y libpq-dev curl build-essential locales libffi-dev libsasl2-dev libldap2-dev \
+                                             dh-autoreconf git python3-dev python3-pip python3-venv python3-wheel && \
+    curl -sL https://deb.nodesource.com/setup_10.x | bash - && \
+    apt-get install -y nodejs && \
+    locale-gen en_US.UTF-8 && export LC_ALL=en_US.UTF-8 && \
+    npm config set registry http://registry.npmjs.org/ && \
+    npm install npm -g && \
+    echo "Running with nodejs:" && node -v && \
+    python3 -m venv /opt/venv && \
+    echo "Running with python:" && /opt/venv/bin/python3 -c 'import platform; print(platform.python_version())' && \
+    /opt/venv/bin/python3 -m pip install --no-cache-dir --upgrade pip setuptools wheel && \
+    /opt/venv/bin/python3 -m pip install --no-cache-dir -e . && \
+    npm install --unsafe-perm && \
+    node_modules/.bin/gulp --cwd /opt/lemur build && \
+    node_modules/.bin/gulp --cwd /opt/lemur package && \
+    npm cache clean --force && \
+    rm -rf node_modules && \
+    python -c 'print(" \033[32m BUILDER DONE \033[0m ")'
+
+#############  APP  #############
+FROM ubuntu:18.04
+LABEL maintainer="Netflix Open Source Development <talent@netflix.com>"
+
+ARG DEBIAN_FRONTEND=noninteractive
+ARG VERSION
+
+ARG URLCONTEXT
+
+ENV uid 1337
+ENV gid 1337
+ENV user lemur
+ENV group lemur
+
+# TODO: do not copy build artefacts
```

Comment:
Did you intend to remove/change this?

Reply:
I pretty sure that builder can be improved more, but at this moment I do not know how I can do it.

Reply:
So, pls do not remove it for future improving.